### PR TITLE
added Admin Display Table page and table component stub

### PR DESCRIPTION
**Pull request by kdeng23**
# Overview

Created an Admin Display Table Page that appears in the navigation bar under the admin tab. The page is an empty stub, where the table component will be added. It is accessible only to admins and renders properly.

# Issues Addressed

#36

### CODE REVIEW

**Review by samar1409 — DISMISSED** [review]
LGTM!

**Review by zjw9635468 — DISMISSED** [review]
LGTM

File: frontend/src/tests/pages/AdminDisplayTablePage.test.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    //     const expectedCommons = {
+    //         name: "My New Commons",
+    //         startingBalance: 500,
+    //         cowPrice: 10,
+    //         milkPrice: 5,
+    //         startDate: '2022-05-12T00:00:00.000Z'
+    //     };
+
+    //     expect(axiosMock.history.post[0].data).toEqual(JSON.stringify(expectedCommons));
+    // });
```
Comment (bzamora020) [inline frontend/src/tests/pages/AdminDisplayTablePage.test.js]: Do we need this commented-out code? 

If the PR is ready to be merged, we might want to remove this

**Review by bzamora020 — CHANGES_REQUESTED** [review]

File: frontend/src/tests/pages/AdminDisplayTablePage.test.js (hunk 1 above)
Comment (kdeng23) [reply frontend/src/tests/pages/AdminDisplayTablePage.test.js]: I've removed the commented-out code.

**Review by zjw9635468 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**